src/main/scala/centaur/test/formulas/TestFormulas.scala

```diff
@@ -67,4 +72,20 @@ object TestFormulas {
       _ <- validateNoCacheHits(metadata, workflow.testName)
     } yield ()
   }
+
+  def runFinalDirsWorkflow(wf: Workflow, dirOption: String): Test[Unit] =  {
+    import spray.json.DefaultJsonProtocol._
+
+    def dirSize = {
+      val options = wf.data.options.get
+      val outputDir = parse(options).toOption.flatMap(_.\\(dirOption).head.asString).get
```

Comment:
re: head & get
you win!  that was my grossness litmus test.  Meant to spur discussion as I contend that this is ok in the testing world.  I can de-gross it if there is consensus otherwise.

`.\\` is an alias for a human readable name, I'll use that instead as you're the second person to